ibis/expr/types/pretty.py

```diff
@@ -227,13 +275,36 @@ def format_dtype(dtype):
     return Text.styled(strtyp, "dim")
 
 
-def to_rich_table(table, console_width=None):
+def to_rich_table(
+    tablish: Table | Column,
+    *,
+    max_rows: int | None = None,
+    max_columns: int | None = None,
+    max_length: int | None = None,
+    max_string: int | None = None,
+    max_depth: int | None = None,
+    console_width: int | float | None = None,
+) -> RichTable:
+    from ibis.expr.types import Column, Table
+
+    if max_rows is None:
+        max_rows = ibis.options.repr.interactive.max_rows
+    if max_columns is None:
+        max_columns = ibis.options.repr.interactive.max_columns
     if console_width is None:
         console_width = float("inf")
+    show_types = ibis.options.repr.interactive.show_types
```

Comment:
should we just deprecate the entire `show_types` option from `ibis.options.repr.interactive`? and always make it true?

Reply:
Let's leave it as is.